.travis.yml

```diff
@@ -28,6 +31,9 @@ before_install:
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid \
     --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1440x900x16"
   - nvm install node
+  - export LANG=de_DE.UTF-8
```

Comment:
I'm thinking we should keep on using a different locale so it at least may catch issues when the code doesn't work across locales.

`# Use a different locale from the default `en_US` to possibly catch errors in code not working across locales (although this does not ensure that the code is independent of all locales).`

@tshradheya What do you think?

Reply:
Maybe we can leave it as a comment in the `travis.yml` file so that it can be checked by developers in cases when huge PR's are being merged. 

Because our original code was written only for `Locale.US` it seems unnecessary to put a different locale in our testing script just in hope to catch error if it fails for that one particular different locale

Reply:
Our original code wasn't written for any locale, it was written without any considerations for any locale which is an issue.

But I guess it is also rather strange, even if the code supports more than one locale (as opposed to being locale independent) we won't include this. I suppose we can remove it altogether.